### PR TITLE
examples: improve the priv-net example

### PR DESCRIPTION
**Pull request by greut**
demo: https://github.com/exoscale/terraform-provider-exoscale/tree/priv-net-links/examples/multi-private-network

### CODE REVIEW

File: examples/multi-private-network/README.md [hunk 1]
```diff
@@ -1,19 +1,57 @@
...
+--- 192.168.0.2 ping statistics ---
+3 packets transmitted, 3 received, 0% packet loss, time 2011ms
+rtt min/avg/max/mdev = 0.590/0.663/0.767/0.075 ms
 ```
 
 Profit!
+
+## Documentation
+
+To understand the ins and outs of what is a [Private network][] in
```
Comment (pyr) [inline examples/multi-private-network/README.md]: To better understand the concept of [Private Networks][] on Exoscale, consult our documentation on [Community][].

Comment (ste-m) [reply examples/multi-private-network/README.md]: id go for consult our [documentation] and skip the term community since we have plans for it.. or not?

Comment (pyr) [reply examples/multi-private-network/README.md]: good too.

**Comment by greut** [comment]
went here: 6d21127e65bddd08b8b3ffb22febbb8be7efe525

**Pull request closed without merging**